<ui version="4.0" >
 <class>SystemViewDlg</class>
 <widget class="QWidget" name="SystemViewDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="lbMenu" >
         <property name="text" >
          <string>Menü:</string>
         </property>
         <property name="buddy" >
          <cstring>lwMenu</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="SimonListWidget" name="lwMenu" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>273</height>
           </size>
          </property>
         </widget>
         <widget class="QTextBrowser" name="tbHelp" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter_2" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QStackedWidget" name="swControls" >
       <widget class="QWidget" name="pgEmpty" >
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600;">Willkommen im Systemmenü&lt;/span>&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Sie befinden sich nun im simon-Systemmenü. Hier können Sie das System konfigurieren.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Bitte beachten Sie, dass falsche Einstellungen den ordentlichen Betrieb von simon behindern bzw. unmöglich machen.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Sollten Sie sich nicht sicher sein, lesen Sie in der betreffenden Hilfe (je Modul; in der linken, unteren Ecke) oder im Handbuch nach oder fragen Sie ihren Administrator um Rat.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Um zu beginnen, wählen Sie bitte ein Untermenü aus der Liste links.&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="pbReset" >
       <property name="text" >
        <string>Zurücksetzen</string>
       </property>
       <property name="icon" >
        <iconset resource="../simon.qrc" >:/images/icons/edit-undo.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbApply" >
       <property name="text" >
        <string>Übernehmen</string>
       </property>
       <property name="icon" >
        <iconset resource="../simon.qrc" >:/images/icons/dialog-ok-apply.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimonListWidget</class>
   <extends>QListWidget</extends>
   <header>simonlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../simon.qrc" />
 </resources>
 <connections/>
</ui>
